src/services/firebase-client/rtdb/index.ts

```diff
@@ -0,0 +1,49 @@
+import firebase from "firebase/app"
+import "firebase/database"
+import { configs as firebaseConfigs } from '../../../constants/firebase'
+
+const isFirebaseInitialized = (): boolean => !!firebase.apps.length
+
+if (!isFirebaseInitialized()) {
+  firebase.initializeApp(firebaseConfigs)
+}
+
+const db = firebase.database()
+
+
+export const create = async (key: string, value: any) => {
+  const error = await db.ref(key).set(value)
+
+  if (error) {
+    console.error('Data could not be saved')
+    console.error(error)
```

Comment:
Not sure if you would like to let UI prompt error to User on error.
Silencing error is a bad practice